src/github.ts

```diff
@@ -149,7 +149,7 @@ export const upload = async (
   const [owner, repo] = config.github_repository.split("/");
   const { name, size, mime, data: body } = asset(path);
   const currentAsset = currentAssets.find(
-    ({ name: currentName }) => currentName == name
+    ({ name: currentName }) => currentName == name.replace(' ','.')
```

Comment:
according to the notes listed [here](https://docs.github.com/en/rest/releases/assets?apiVersion=2022-11-28#upload-a-release-asset) we may want to expand this to include other characters but I'm okay with this as is for now. 

The GitHub docs don't provide a non ambiguous list of characters that it transforms to periods. The best I could do is manually upload release asses with files that contain a wide variety of characters to learn from example